# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/usbutils/linux-mod_devicetable_h.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur usbutils-0.11/libusb/access.c usbutils-0.11-p/libusb/access.c
--- usbutils-0.11/libusb/access.c	1999-12-22 15:14:16.000000000 +0000
+++ usbutils-0.11-p/libusb/access.c	2004-10-27 18:08:16.175682408 +0000
@@ -31,6 +31,8 @@
 #include "libusb.h"
 
 #ifdef HAVE_LINUX_USB_H
+#include <linux/types.h>
+typedef unsigned long kernel_ulong_t;
 #include <linux/usb.h>
 #else
 #include "usb.h"
diff -dur usbutils-0.11/libusb/descparse.c usbutils-0.11-p/libusb/descparse.c
--- usbutils-0.11/libusb/descparse.c	1999-12-22 15:14:16.000000000 +0000
+++ usbutils-0.11-p/libusb/descparse.c	2004-10-27 18:08:54.696826304 +0000
@@ -31,6 +31,8 @@
 #include "libusb.h"
 
 #ifdef HAVE_LINUX_USB_H
+#include <linux/types.h>
+typedef unsigned long kernel_ulong_t;
 #include <linux/usb.h>
 #else
 #include "usb.h"
